src/groundStation/system.py

```diff
@@ -645,10 +643,194 @@ def __init__(self):
+                        'subPort': 45,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b',
+                                'Secure Key': '>u4',
+                            }
+                        }
+                    },
                 }
             },
+            
+            'CONFIGURATION': {
+                'port': 9,  # As per EPS docs
+                'subservice': {
+                    'GET_ACTIVE_CONFIG': {
+                        'what': 'Gets config values in active mode for a specific type',
+                        'subPort': 0,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #In command parser it gets the data type from the return value for 'type'
+                            }
+                        }
+                    },   
+                    'GET_MAIN_CONFIG': {
+                        'what': 'Gets config values in main mode for a specific type',
+                        'subPort': 1,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #See comment for active
+                            }
+                        }
+                    },  
+                    'GET_FALLBACK_CONFIG': {
+                        'what': 'Gets config values in fallback mode for a specific type',
+                        'subPort': 2,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #See comment for active
+                            }
+                        }
+                    },
+                    'GET_DEFAULT_CONFIG': {
+                        'what': 'Gets config values in default mode for a specific type',
+                        'subPort': 3,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #See comment for active
+                            }
+                        }
+                    },                                                                                                                      
+                    'SET_CONFIG': {
```

Comment:
Woucl it be possible to get scripted tests for these too

Reply:
Sure, but since we won't change the EPS's software, what would that test script do? I think a simple command in CLI would be enough to make sure EPS is responding.

Reply:
We should make sure that every command here does what we expect it to.

Reply:
For services that communicate with another firmware, I was thinking that we should rethink testing everything. For example, if we hard reset EPS, it will erase all the saved configs. Would we want to have that in the test script? Or changing a certain power channel could shut off a subsystem. The same thing goes with time-variant parameters. There's no way you can read the uptime from UHF or EPS for example and have an expectation.

Reply:
Okay maybe we'll leave the configs alone. But we should absolutely test toggling power, then checking HK to see if it's off. We could add the ability to verify that the time is within some reasonable range.

Reply:
We just really don't want to find out during the mission that these commands don't behave the way we think they do

Reply:
@arrooney I'll keep this in mind (feel free to add a ticket) but I think it's better discussed later and added in another PR. 
The varTypes dictionary was also added and commandparser is tidier now. Any other comments?